velox/tpch/gen/dbgen/build.cpp

```diff
@@ -85,10 +86,22 @@ static void gen_phone(DSS_HUGE ind, char* target, seed_t* seed) {
   RANDOM(exchg, 100, 999, seed);
   RANDOM(number, 1000, 9999, seed);
 
-  sprintf(target, "%02d", static_cast<int>(10 + (ind % NATIONS_MAX)));
-  sprintf(target + 3, "%03d", static_cast<int>(acode));
-  sprintf(target + 7, "%03d", static_cast<int>(exchg));
-  sprintf(target + 11, "%04d", static_cast<int>(number));
+  int res = sprintf(target, "%02d", static_cast<int>(10 + (ind % NATIONS_MAX)));
```

Comment:
Why not use fmt::format instead of the complications with sprintf  ?

Reply:
Done, but still we need to convert std::string back to c style right and store the output in target.